src/patternfly/upgrade-examples/BadgeUpgradeExamples/badge-upgrade-examples.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-d-badge-upgrade-examples {{pf-d-badge-upgrade-examples--modifier}}"
```

Comment:
I don't think we need this for upgrade examples. Looks like it's a holdover from demos (`.pf-d-`). It isn't even in use on demos, except for the LoginPage demo, and I think it can be removed on that one, too.

Reply:
agreed, i just left in {{> @partial-block}}